### PR TITLE
Turnstile update

### PR DESCRIPTION
**Pull request by kityansiu**
Update Turnstile so that this example fulfills the following purposes:
- ontology overlay exemplar 
- CDR usage exemplar
- instance data exemplar
- an exemplar ingestion package with a Load-TurnstileData.sh script

Following the other overlay naming conventions, create a GE-Ontology directory with the turnstile overlay, and create a separate directory with the turnstile instance data.

This update will be piecemeal. In the meantime, the original Turnstile-Ontology will be preserved until the example is fully migrated.

Related issues: #224, #440, #457

### CODE REVIEW

**Comment by Robert-Adelard** [comment]
Issue #457 may also be relevant - please take a look.

**Comment by Robert-Adelard** [comment]
Issue #480 may also be relevant - please take a look.

**Comment by Robert-Adelard** [comment]
Also addresses #487 - the new URI for the Turnstile example will be `http://arcos.turnstile/GE`.

**Comment by Robert-Adelard** [comment]
See also #506, which might be relevant.

File: GE-Ontology/ontology/GE.sadl [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+        turnstile:wasInformedBy of ControlCouplingAnalysis has values of type SoftwareComponentTestExecution.
+
+        //Model Location: DevelopmentPlan.04-SoftwareVerification.04-SoftwareAnalysis.DataCouplingAnalysis
+      DataCouplingAnalysis is a type of ANALYSIS.
+        turnstile:wasInformedBy of DataCouplingAnalysis has values of type SoftwareComponentTestExecution.
+
+        //Model Location: DevelopmentPlan.04-SoftwareVerification.04-SoftwareAnalysis.StructualCoverageAnalysis
+      StructualCoverageAnalysis is a type of ANALYSIS.
+        CoveredNodes describes StructualCoverageAnalysis with values of type int.
+        UncoveredNodes describes StructualCoverageAnalysis with values of type int.
```
Comment (glguy) [inline GE-Ontology/ontology/GE.sadl:181]: These two properties should be lowercased

Comment (kityansiu) [reply GE-Ontology/ontology/GE.sadl:181]: Thanks for catching these! Will modify.

**Comment by Robert-Adelard** [comment]
See also #556

**Pull request merged**